lms/djangoapps/shoppingcart/models.py

```diff
@@ -980,6 +981,17 @@ class InvoiceTransaction(TimeStampedModel):
     created_by = models.ForeignKey(User)
     last_modified_by = models.ForeignKey(User, related_name='last_modified_by_user')
 
+    @classmethod
+    def get_invoice_transaction(cls, invoice_id):
```

Comment:
Nice-to-have: These should really be manager methods: https://docs.djangoproject.com/en/1.4/topics/db/managers/. It doesn't need to be done now; but, keep it in mind for future reference. It's a nice way to keep the model clean and trim.